lib/flow.ml

```diff
@@ -220,25 +220,27 @@ module Make (Flow : Mirage_flow.S) = struct
       let pad = Cstruct.create (len - 8) in
       Cstruct.append pad seq
     in
-    xor nonce seq
+    Cstruct.to_string (xor nonce seq)
```

Comment:
I guess if this is performance critical it could be revised and cstruct could be avoided competely (within the entire module).

Reply:
Yes, I'm a bit lazy to change this :+1:. Also it's about the `Mirage_flow.S` implementation which will disappear one time once and for all one day :wink:. The goal is mainly focus on `bob` which has its own implementation.